_includes/footer.html

```diff
@@ -1,63 +1,45 @@
-                      <li class="footer-subnav-item"><a class="footer-subnav link" href="about/guest_artists">Guest Artists</a></li>
-                      <li class="footer-subnav-item"><a class="footer-subnav link" href="/about/board">Board of Directors</a></li>
-                  </ul>
-              </li>
-              <li class="footer-nav-item">
-                  <a class="footer-nav-link" href="/past-productions">Past Productions</a>
-              </li>
-              <li class="footer-nav-item">
-                  <a class="footer-nav-link" href="/sponsors">Sponsors</a>
-                  <ul class="footer-subnav">
-                      <li class="footer-subnav-item"><a class="footer-subnav-link" href="/sponsors#current">Current</a></li>
-                      <li class="footer-subnav-item"><a class="footer-subnav-link" href="/sponsors#new">Become A Sponsor</a></li>
-                  </ul>
-              </li>
-              <li class="footer-nav-item"><a class="footer-nav-link" href="/donate">Donate</a>
-                  <ul class="footer-subnav">
-                      <li class="footer-subnav-item"><a class="footer-subnav-link" href="/donate">One-Time Donation</a></li>
-                      <li class="footer-subnav-item"><a class="footer-subnav-link" href="/donate">Recurring Donation</a></li>
-                  </ul>
-              </li>
-          </ul>
-          <div class="footer-auxnav">
-              <div class="newsletter">
-                  <div class="newsletter-inputarea">
-                      <div id="mc_embed_signup">
-                          <form id="mc-embedded-subscribe-form" class="validate" action="//accidentalshakespeare.us8.list-manage.com/subscribe/post?u=62e153225217fde3b5c71b8f9&amp;id=77a38c823e" method="post" name="mc-embedded-subscribe-form" target="_blank">
-                              <div id="mc_embed_signup_scroll">
-                                  <div class="mc-field-group">
-                                      <label class="required email">Email<input id="mce-EMAIL" class="required email" type="email" name="EMAIL" placeholder="Enter Email" value="" size="35" /></label>
-                                      
-                                      <input id="mc-embedded-subscribe" class="button" type="submit" name="subscribe" value="Subscribe" />
-                                  </div>
-                              </div>
-                          </form>
-                      </div>
-                  </div>
-              </div>
+<footer id="myFooter">
+  <div class="container">
+    <div class="row">
+      <!--<div class="col-sm-3">
+                    <h2 class="logo"><a href="#"> LOGO </a></h2>
+                </div>-->
+      <div class="col-sm-3 footerCol">
+        <h5>About us</h5>
+        <ul>
+          <li><a href="/about">Company / Staff</a></li>
+          <li><a href="/about/guest_artists">Guest Artists</a></li>
```

Comment:
This links have been broken in almost every metric since we started. I don't think we're getting new about pages and such.